_schema.graphql

```diff
@@ -5381,6 +5381,27 @@ type EndSalePayload {
   clientMutationId: String
 }
 
+# A user-readable error
+type Error {
```

Comment:
This seems like a pretty generic thing added in Gravity. Do we want to rename it here?